### ISSUE TITLE
OpenResty 1.4.2.* release?

### ISSUE THREAD

**Issue by erikcw** [leafo/heroku-openresty#3]
It would be great to have the 1.4.2 branch of OpenResty available.  Is there anything blocking adding it?

**Comment by leafo** [comment]
Nope, I just haven't built the latest version recently. I'll do shortly

**Comment by erikcw** [comment]
Actually -- it would be interesting to have a wiki entry or some other documentation describing the build process so users can role their own builds.

**Comment by leafo** [comment]
https://github.com/leafo/heroku-openresty/tree/master/support
